pkg/pillar/types/locationconsts.go

```diff
@@ -4,21 +4,62 @@
 package types
 
 const (
-	appImgObj = "appImg.obj"
-
-	// TmpDirname - Temporary dir for zededa components
+	// TmpDirname - temporary dir. for agents to use.
 	TmpDirname = "/var/tmp/zededa"
-	// PersistDir - Persist Directory
+	// AADirname - directory for per-model Assignable Adapters files
+	AADirname = TmpDirname + "/AssignableAdapters"
+	// DNCDirname - directory for per-model Network Config files
+	DNCDirname = TmpDirname + "/DeviceNetworkConfig"
+
+	// PersistDir - Location to store persistent files.
 	PersistDir = "/persist"
-	// PersistRktDataDir - persist rkt dir used by rkt
-	PersistRktDataDir = PersistDir + "/rkt"
-	// RwImgDirname - location for images
-	RwImgDirname = PersistDir + "/img" // We store images here
-	// DownloadDirname - Location for downloader to download images
+	// RwImgDirname - We store images here
```

Comment:
Only used  by domainmgr, hence not common.

Reply:
moved it back to domainmgr